app/controllers/monitor_controller.rb

```diff
@@ -0,0 +1,21 @@
+class MonitorController < ApplicationController
+  def create
+    monitor = LinkMonitor::UseCase::CreateResourceMonitor.new(links: permitted_params[:links], service: permitted_params[:service]).call
+
+    if monitor.valid?
+      render(json: monitor_report(monitor), status: 200)
+    else
+      render(json: monitor.errors, status: 422)
```

Comment:
In other controllers we've just called `monitor.validate!` and left Rails to handle the output for us. Perhaps we can do the same here?

Reply:
👍